stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -14,13 +14,15 @@
 // limitations under the License.
 
 #include <vector>
+#include <utility>
```

Comment:
Whole file needs `clang-format`, but I'd just format the additions to keep the diff small.